apps/src/lib/util/StatsigReporter.js

```diff
@@ -26,13 +39,15 @@ class StatsigReporter {
     const options = {
       environment: {tier: getEnvironment()},
       localMode: this.local_mode,
+      disableErrorLogging: true,
     };
-    this.initialize(api_key, options);
+    this.initialize(api_key, user, options);
   }
 
-  async initialize(api_key, options) {
+  // Initialize with a null user object- current user redux will update on sign in
```

Comment:
what does this comment mean sorry?

Asking because I do like the idea of comment here that this will potentially update the statsig user :)

Reply:
Ah I hadn't updated it yet- it's updated and accurate now!